src/Umbraco.Web.UI/Umbraco/PartialViewMacros/Templates/Login.cshtml

```diff
@@ -8,7 +8,8 @@
 
 @{
     var loginModel = new LoginModel();
-    
+    loginModel.RedirectUrl = HttpContext.Current.Request.Url.AbsolutePath;

```

Comment:
This works for me — but I don't know if `HttpContext.Current.Request.Url.AbsolutePath` is the correct way to get the desired value here?